tests/unit/replica-cob-usage.tcl

```diff
@@ -0,0 +1,32 @@
+proc client_field {master f} {
+    set slave [$master client list type slave]
+    if {![regexp $f=(\[a-zA-Z0-9-\]+) $slave - res]} {
+        error "field $f not found for in $slave"
+    }
+    return $res
+}
+
+start_server {tags {"wait external:skip"}} {
+start_server {} {
+    set slave [srv 0 client]
+    set slave_host [srv 0 host]
+    set slave_port [srv 0 port]
+    set master [srv -1 client]
+    set master_host [srv -1 host]
+    set master_port [srv -1 port]
+
+    test "replica client output buffer usage: client output buffer usage shouldn't include the size of the data it has already consumed" {
```

Comment:
since the main concern we have is about obuf limit disconnection, maybe we should test that instead of just the info metric?
i.e. it could happen that one day the code affecting the obuf limit disconnection is change to break this concern, but he info metric remains unaffected and this test will still pass.